synthetics_build-reports.tf

```diff
@@ -0,0 +1,29 @@
+resource "datadog_synthetics_test" "docker_404" {
+  type = "api"
+  request_definition {
+    method = "GET"
+    url    = "https://builds.reports.jenkins.io/build_status_reports/infra.ci.jenkins.io/docker-jobs/docker-404/main/status.json"
+  }
+  assertion {
+    type     = "statusCode"
+    operator = "is"
+    target   = "200"
+  }
+  assertion {
+    type     = "body"
+    operator = "contains"
+    target   = "\"build_status\":\"SUCCESS\""
```

Comment:
Could you change the operator to use the `jsonpath` as per https://docs.datadoghq.com/synthetics/api_tests/http_tests/?tab=requestoptions#define-assertions instead?

It avoid tedious pattern matching with escaped quotes